packages/react/src/modules/feed/components/NotificationCell/NotificationCell.tsx

```diff
@@ -23,91 +35,129 @@ type BlockByName = {
+
+        return maybeNavigateToUrlWithDelay(button.action);
+      },
+      [onButtonClick, feedClient, item],
+    );
+
+    const onKeyDown = React.useCallback(
+      (ev: React.KeyboardEvent<HTMLDivElement>) => {
+        switch (ev.key) {
+          case "Enter": {
+            ev.stopPropagation();
+            onContainerClickHandler();
+            break;
+          }
+          default:
+            break;
         }
-        default:
-          break;
-      }
-    },
-    [onClick],
-  );
-
-  const actor = item.actors[0];
-
-  return (
-    <div
-      ref={ref}
-      className={`rnf-notification-cell rnf-notification-cell--${colorMode}`}
-      onClick={onClick}
-      onKeyDown={onKeyDown}
-      tabIndex={0}
-    >
-      <div className="rnf-notification-cell__inner">
-        {!item.read_at && <div className="rnf-notification-cell__unread-dot" />}
-
-        {renderNodeOrFallback(
-          avatar,
-          actor && "name" in actor && actor.name && (
-            <Avatar name={actor.name} src={actor.avatar} />
-          ),
-        )}
-
-        <div className="rnf-notification-cell__content-outer">
-          {blocksByName.body && (
-            <div
-              className="rnf-notification-cell__content"
-              dangerouslySetInnerHTML={{ __html: blocksByName.body.rendered }}
-            />
+      },
+      [onContainerClickHandler],
+    );
+
+    const actor = item.actors[0];
+
+    return (
+      <div
+        ref={ref}
+        className={`rnf-notification-cell rnf-notification-cell--${colorMode}`}
+        onClick={onContainerClickHandler}
+        onKeyDown={onKeyDown}
+        tabIndex={0}
+      >
+        <div className="rnf-notification-cell__inner">
+          {!item.read_at && (
+            <div className="rnf-notification-cell__unread-dot" />
           )}
 
-          {children && (
-            <div className="rnf-notification-cell__child-content">
-              {children}
-            </div>
+          {renderNodeOrFallback(
+            avatar,
+            actor && "name" in actor && actor.name && (
+              <Avatar name={actor.name} src={actor.avatar} />
+            ),
           )}
 
-          <span className="rnf-notification-cell__timestamp">
-            {formatTimestamp(item.inserted_at, { locale })}
-          </span>
+          <div className="rnf-notification-cell__content-outer">
+            {blocksByName.body && (
+              <div
+                className="rnf-notification-cell__content"
+                dangerouslySetInnerHTML={{
+                  __html: (blocksByName.body as MarkdownContentBlock).rendered,
+                }}
+              />
+            )}
+
+            {buttonSet && (
+              <div className="rnf-notification-cell__button-group">
+                <ButtonGroup>
+                  {buttonSet.buttons.map((button, i) => (
+                    <Button
+                      variant={i === 0 ? "primary" : "secondary"}
```

Comment:
```suggestion
                      variant={button.name}
```
Alternatively, buttons should have a name field set to either "primary" or "secondary".

Reply:
Ah got it, because the variant for `Button` component will still be "secondary" even for the third/tertiary button.